catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/impl/OpenSearchParserImplTest.java

```diff
@@ -294,7 +151,6 @@ public void populateContextualEmptyMap() {
                 .split(",")));
     String urlStr = webClient.getCurrentURI().toString();
     assertThat(urlStr, not(containsString("q=")));
-    assertThat(urlStr, containsString(OpenSearchParserImpl.SEARCH_TERMS));
```

Comment:
These missing search terms will be fixed in the next release. I'll add the ticket number here when I find it.

Reply:
@emmberk did you find that ticket number

Reply:
Not yet. I'm working on it now.